javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/AbstractLexicalPreservingTest.java

```diff
@@ -76,22 +77,22 @@ protected void assertTransformed(String exampleName, Node node) throws IOExcepti
         String actualCode = LexicalPreservingPrinter.print(node);
 
         // Note that we explicitly care about line endings when handling lexical preservation.
-        assertEqualsString(expectedCode, actualCode);
+        TestUtils.assertEqualsStringIgnoringEol(expectedCode, actualCode);
```

Comment:
You are changing behavior that is intentional!

> Note that we explicitly care about line endings when handling lexical preservation.

Reply:
Then I did not get your comment https://github.com/javaparser/javaparser/pull/3361#issuecomment-911969873 right. Reverted bbfc0d7987e9659bef6b7206c.

Reply:
Apologies, without context (note the comment on line 79) it wasn't obvious from the quoted errors that this was a place where line endings were important.